test/kw/kw_apis.h

```diff
@@ -53,68 +57,219 @@ TEST(kw, renew_token_correct)
 
 TEST(kw, login_correct)
```

Comment:
According to https://github.com/google/googletest/blob/master/googletest/docs/primer.md#simple-tests test names should not contain `_`. So this would be `TEST(kw, loginCorrect)` or `TEST(kw, LoginCorrect)`

Reply:
Good spot here.